lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset.py

```diff
@@ -498,6 +507,9 @@ def exec_module(self, **kwargs):
             # Set default location
             self.location = resource_group.location
 
+        if self.custom_data:
+            self.custom_data = to_native(base64.b64encode(to_bytes(self.custom_data)))
```

Comment:
from code, it's  script content, but i think url or filename should be supported for long script.

Reply:
make sense